README.md

```
@@ -0,0 +1,28 @@
+- 사용자 페이지: [localhost:8080/](http://localhost:8080/)
+    ```plaintext
+  테스트 사용자 계정
+  email: sooyang@example.com
+  password: 1234
+   ```
+
+## 기능 구현 목록
+
+- [x] 화면 수정 - 결제 UI (위젯) 추가
+- [x] 기능 수정
+    - [x] 에약 시 결제 단계 추가 - 사용자가 결제를 완료해야 예약할 수 있도록 수정
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)